internal/k8s.go

```diff
@@ -61,3 +64,32 @@ func OutputK8sCreds(creds *clientauthv1beta1.ExecCredential, cacheKey string) {
 	os.Stdout.Write(json)
 	os.Stdout.WriteString("\n")
 }
+
+func PingK8sCreds(creds *clientauthv1beta1.ExecCredential, tenantID string) error {
+	config := &rest.Config{
+		Host: creds.Spec.Cluster.Server,
+		TLSClientConfig: rest.TLSClientConfig{
+			Insecure: true,
+		},
+		BearerToken: creds.Status.Token,
+	}
+
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+
+	namespace := tenantID
```

Comment:
the tenant ID at this point is always a GUID and a not a name, so this won't work.  you'll need to get the tenant name elsewhere.

Reply:
@joek-duplo Added the logic to get the tenant name from the tenant ID. If necessary, I could add an optimization to cache the tenant flag as the tenant name before it is converted to a tenant ID. It would save a call to the Duplo API, but there's no benefit of course if the ID was provided in the flag in the first place. I am biasing toward simplicity for the time being on that one.

Of note, I also moved the creation of the cache key for k8s creds on tenants to be performed after the name has been determined to so that the cache key for a tenant on k8s will consistently be based on the host and tenant name.